pysat/instruments/pysat_testing2d_xarray.py

```diff
@@ -4,25 +4,37 @@
 """
 from __future__ import print_function
 from __future__ import absolute_import
-import os
 
+import functools
 import numpy as np
-import pandas as pds
 import xarray as xr
 
 import pysat
-from pysat.instruments.methods import testing as test
+from pysat.instruments.methods import testing as mm_test
 
 platform = 'pysat'
 name = 'testing2D_xarray'
 
 pandas_format = False
+_test_dates = {'': {'': pysat.datetime(2009, 1, 1)}}
 
 
 def init(self):
+    """ Initialization function
```

Comment:
There are more complete versions of these docstrings in the template instruments.